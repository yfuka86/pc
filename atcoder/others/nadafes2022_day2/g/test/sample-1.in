3 10
1 9
4 8
7 10
