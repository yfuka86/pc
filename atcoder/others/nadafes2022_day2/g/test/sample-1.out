151
20
3
3
1
0
1
1
0
0
